.github/workflows/black.yml

```diff
@@ -0,0 +1,19 @@
+name: Black
+
+on: [push, pull_request]
+
+jobs: 
+  Black:
+    runs-on: ubuntu-latest
+    container: fedora:32
+    steps:
+      - name: Install Pipenv and Git
+        run: dnf install -y pipenv git 
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup environment
+        run: pipenv sync --dev 
+      - name: Run Black
+        uses: psf/black@stable
+        with: 
+          args: ". --check --skip-string-normalization"
```

Comment:
I think it will be better to run it with the Pipfile content:
```
run: pipenv run python -m black . --check --skip-string-normalization
```
Maybe even try the direct one: `pipenv run black . --check --skip-string-normalization`

It will be in sync with what you expect to run locally.

Reply:
I replaced lines 17-19 with both of your suggestions and for some reason now the CI is not triggered at all after force pushing as it did. It also happened to us with flake8 at the time, but I'm not sure how to solve it, it had something to do with new changes or something but I really can't find a solution for this.

Reply:
Isn't it because there are conflicts now?

Reply:
oh ok, didn't notice..

Reply:
@Yarboa @Leeadar it seems the PyTest fails for the changes of the merged PR again on the same error, isn't it was supposed to be fixed already?

Reply:
It is most likely your rebase that caused it.
Try to run the tests without your last commit locally. And re-do the formatting (the last commit) from scratch and test.

Reply:
Finally, thanks, great idea @EdDev .